src/lib/runtime_config/runtime_config.ml

```diff
@@ -48,6 +60,15 @@ let of_yojson_generic ~fields of_yojson json =
   dump_on_error json @@ of_yojson
   @@ yojson_strip_fields ~keep_fields:fields json
 
+let rec map_results ~f = function
```

Comment:
@nholland94  The PR to develop was already merged in a week ago. Does that now negate these comments? Thanks.

Reply:
Thanks @nholland94 . That layer (mina_stdlib) doesn't seem available in compatible. Is it OK just to copy the implementation into the runtime_config.ml, or were you thinking of something else.

Reply:
It's fine to do that for compatible, but would be great if you could put it in it's proper place when this is merged up to other branches. It's annoying right now since the branches are very different, but we will be retiring compatible soon.